exo.kernel.commons/src/main/java/org/exoplatform/commons/debug/ObjectDebuger.java

```diff
@@ -30,8 +30,8 @@
 /**
  * Jul 29, 2004
  * 
- * @author: Tuan Nguyen
- * @version: $Id: ObjectDebuger.java,v 1.4 2004/09/21 00:04:43 tuan08 Exp $
+ * @author Tuan Nguyen
+ * @version $Id: ObjectDebuger.java,v 1.4 2004/09/21 00:04:43 tuan08 Exp $
```

Comment:
Does these modication are necessary for jdk 17 ? 
If not, we should remove it as it make the PR more complex to read